src/cascadia/TerminalSettingsEditor/ProfileViewModel.cpp

```diff
@@ -352,6 +352,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
     void ProfileViewModel::SetupAppearances(Windows::Foundation::Collections::IObservableVector<Editor::ColorSchemeViewModel> schemesList, Editor::IHostedInWindow windowRoot)

```

Comment:
Do we want to rename this to something like `SetupHelper` or something since it technically isn't _just_ setting up the appearances anymore? I don't feel too strongly about it

Reply:
If you're going to call it from outside the class, `Helper` is not a good name.

For consideration: a hosting window might reasonably be considered part of the _view_ moreso than part of the view model. Can we just pass it in when the view is created? It doesn't change over time, and does not have any xaml bound to it.

Reply:
Makes sense! Made the changes